### PR TITLE
Change FeedbackRequest for UI

### PR DESCRIPTION
**Pull request by Farleo**
add assign question to FBRequest

### CODE REVIEW

File: src/main/java/ita/softserve/course_evaluation/repository/QuestionRepository.java [hunk 1]
```diff
@@ -2,9 +2,18 @@
...
+import java.util.List;
+
 @Repository
 public interface QuestionRepository extends JpaRepository<Question,Long> {
+	
+	@Query("SELECT questions FROM FeedbackRequest f LEFT JOIN f.questions questions where f.id= :id")
+	List<Question> findAllQuestionsByFeedbackRequest(Long id);
+	
+	@Query("SELECT questions.id FROM FeedbackRequest f LEFT JOIN f.questions questions where f.id= :id")
+	List<Long> findAllQuestionIdsByFeedbackRequest(Long id);
```
Comment (VovaProtsiv) [inline src/main/java/ita/softserve/course_evaluation/repository/QuestionRepository.java:18]: Use native query

**Review by VovaProtsiv — APPROVED** [review]

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestQuestionController.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	@ApiResponses(value = {
+			@ApiResponse(code = 200, message = HttpStatuses.OK, response = QuestionDto.class),
+			@ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+			@ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+			@ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+	})
+	@GetMapping("/{id}/questions")
+	public ResponseEntity<List<QuestionDto>> getQuestionsByFeedbackRequest(@PathVariable("id") Long feedbackRequestId) {
+		final List<QuestionDto> questions = feedbackRequestQuestionService.getQuestionsByFeedbackRequest(feedbackRequestId);
+		return questions != null && !questions.isEmpty()
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestQuestionController.java:42]: ```CollectionUtils.isEmpty()```

File: src/main/java/ita/softserve/course_evaluation/service/impl/FeedbackRequestQuestionServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	
+	public FeedbackRequestQuestionServiceImpl(QuestionRepository questionRepository, FeedbackRequestRepository feedbackRequestRepository) {
+		this.questionRepository = questionRepository;
+		this.feedbackRequestRepository = feedbackRequestRepository;
+	}
+	
+	@Override
+	public List<QuestionDto> getQuestionsByFeedbackRequest(Long id) {
+		List<QuestionDto> questionDtoList = QuestionDtoMapper.toDto(questionRepository.findAllQuestionsByFeedbackRequest(id));
+		return questionDtoList.isEmpty() ? new ArrayList<>() : questionDtoList;
```
Comment (Forfend) [inline src/main/java/ita/softserve/course_evaluation/service/impl/FeedbackRequestQuestionServiceImpl.java:29]: just ```return questionDtoList```

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestQuestionController.java (hunk 2 above)
Comment (Farleo) [reply src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestQuestionController.java:42]: ok

File: src/main/java/ita/softserve/course_evaluation/repository/QuestionRepository.java (hunk 1 above)
Comment (Farleo) [reply src/main/java/ita/softserve/course_evaluation/repository/QuestionRepository.java:18]: ok

**Review by VovaProtsiv — APPROVED** [review]

**Pull request merged**